terraform/telemetry.py

```diff
@@ -0,0 +1,39 @@
+from google.cloud import pubsub_v1
+from datetime import datetime
+import sys
+
+project_id = "stackdriver-sandbox-230822"
+topic_id = "telemetry"
+
+publisher = pubsub_v1.PublisherClient()
+# The `topic_path` method creates a fully qualified identifier
+# in the form `projects/{project_id}/topics/{topic_id}`
+topic_path = publisher.topic_path(project_id, topic_id)
+
+# define the following helper functions
+def get_datetime_str():
+    # send current date and time for end of installation script
+    now = datetime.now()
+    data = now.strftime("%m/%d/%Y %H:%M:%S")
+    return data
+
+def send_message(args, debug=False):
+    # send in json format
+    data = "{\n\"date\":\"" + get_datetime_str() + "\","
```

Comment:
you should define your message format using proto
https://developers.google.com/protocol-buffers/docs/pythontutorial
or you can use json